lib/sidekiq.rb

```diff
@@ -184,11 +185,20 @@ def self.dump_json(object)
     JSON.generate(object)
   end
 
+  class << self
+    attr_accessor :logger_formatter
+  end
+
   def self.logger
-    Sidekiq::Logging.logger
+    @logger ||= Sidekiq::Logger.new(STDOUT, level: Logger::INFO)
   end
-  def self.logger=(log)
-    Sidekiq::Logging.logger = log
+
+  def self.logger=(logger)
+    @logger = logger
+  end
+
+  def self.tid
```

Comment:
Technically it's thread-specific.  I'm not sure it belongs in the Sidekiq module or perhaps Sidekiq::Util would be more appropriate since it's an internal impl, not an API designed to be used by app code.